dask/dataframe/core.py

```diff
@@ -2743,6 +2764,17 @@ def _repr_html_(self):
     Series._bind_comparison_method(name, meth)
 
 
+def is_broadcastable(dfs, s):
+    """
+    This Series is broadcastable against another dataframe in the sequence
+    """
+    return (isinstance(s, Series) and
+            s.npartitions == 1 and
+            s.known_divisions and
+            any(s.divisions == (min(df.columns), max(df.columns))
```

Comment:
This seems fine to me. Just for posterity, I was initially worried about three things:
- What happens when columns is an unordered `CategoricalIndex`
- What happens when columns is a `MultiIndex`
- Performance of using `min(columns)` instead of `columns.min()`

The first one actually works fine, since the iterator goes over the values in the index not the categorical codes (makes sense).  For multiindex, the iterator is just tuples, so that also works. The third is interesting - the performance of `min` is waaaay faster than the `min` method (not that it matters here, columns is likely to be small). This is true for `Index` and `MultiIndex`.

```python
In [62]: ind = pd.Index(map(str, range(100000)))

In [63]: %timeit ind.min()
100 loops, best of 3: 10.8 ms per loop

In [64]: %timeit min(ind)
100 loops, best of 3: 3.16 ms per loop

In [65]: ind = pd.Index(['a', 'b', 'c', 'd', 'e'])  # Something smaller

In [66]: %timeit ind.min()
The slowest run took 4.64 times longer than the fastest. This could mean that an intermediate result is being cached.
10000 loops, best of 3: 30.2 µs per loop

In [67]: %timeit min(ind)
The slowest run took 43.95 times longer than the fastest. This could mean that an intermediate result is being cached.
1000000 loops, best of 3: 1.62 µs per loop
```